farmerbot/internal/farmerbot_test.go

```diff
@@ -32,6 +32,7 @@ func TestFarmerbot(t *testing.T) {
 	inputs := Config{
 		FarmID:        1,
 		IncludedNodes: []uint32{1, 2},
+		PriorityNodes: []uint32{2},
```

Comment:
I think we need multiple validations for priority nodes field e.g duplicated record, invalid node id, missing node id, node id that doesn't belong to a farm, .. etc

Reply:
to be added: https://github.com/threefoldtech/tfgrid-sdk-go/issues/1017